### PR TITLE
Light edit of Retry Services section

### PR DESCRIPTION
**Pull request by martinduke**
Fixes https://github.com/martinduke/draft-duke-quic-load-balancers/issues/60

### CODE REVIEW

File: draft-ietf-quic-load-balancers.md [hunk 1]
```diff
@@ -526,30 +526,33 @@ trusted network service. One requires no shared state; the server need only be
...
+of mode MAY be made on a per-packet or per-connection basis, through a
+stochastic process or based on client address.
 
 A retry service MUST forward all packets for a QUIC version it does not
 understand. Note that if servers support versions the retry service does not,
-this may unacceptably increase loads on the servers. However, dropping these
-packets would introduce chokepoints to block deployment of new QUIC versions.
-Note that future versions of QUIC might not have Retry packets, or require
-different information.
+this may increase loads on the servers. However, dropping these packets would
```
Comment (nibanks) [inline draft-ietf-quic-load-balancers.md]: Maybe something more like "may lead to increased load on the server".

File: draft-ietf-quic-load-balancers.md [hunk 2]
```diff
@@ -526,30 +526,33 @@ trusted network service. One requires no shared state; the server need only be
 configured to trust a retry service, though this imposes other operational
 constraints. The other requires shared key, but has no such constraints.
 
-Retry services MUST forward all non-Initial QUIC packets, as well as Initial
```
Comment (nibanks) [inline draft-ietf-quic-load-balancers.md]: Why was this removed?

File: draft-ietf-quic-load-balancers.md (hunk 1 above)
Comment (martinduke) [reply draft-ietf-quic-load-balancers.md]: I prefer the more concise sentence.

File: draft-ietf-quic-load-balancers.md (hunk 2 above)
Comment (martinduke) [reply draft-ietf-quic-load-balancers.md]: I was going to rewrite this to allow Retry services to drop 0RTT too, and then realized that this line allows unlimited packets from attackers if they are non-initial, non-0RTT.

This seemed bad, but now I see that because short packets could very easily involve a CID or IP address change, there is no way for the service to filter them effectively. I'll restore it.
